spec/linter-phpcs-spec.js

```diff
@@ -48,9 +48,11 @@ describe('The phpcs provider for Linter', () => {
         lint(editor).then(messages => {
           expect(messages[0].type).toBeDefined();
           expect(messages[0].type).toEqual('ERROR');
-          expect(messages[0].text).toBeDefined();
-          expect(messages[0].text).toEqual('TRUE, FALSE and NULL must be ' +
-            'lowercase; expected "true" but found "TRUE"');
+          expect(messages[0].html).toBeDefined();
+          expect(messages[0].html).toEqual('' +
```

Comment:
Change this to `toBe`.

`toEqual()` is the same as `==`, `toBe()` is the same as `===`.